ci/check-clang-format-version.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+
+EXPECTED_CLANG_VERSION = "18.1.6"
+
+
+def main():
+    result = subprocess.run(("clang-format", "--version"), capture_output=True)
+    result.check_returncode()
+
+    version_str = result.stdout.decode("utf-8").split(" ")[2].strip()
```

Comment:
nit: the default split character is space, so you can just do: `.split()`

Reply:
Thanks. Updated in de4d3d8